keymaster/ta/mbedtls_proxy.c

```diff
@@ -736,8 +737,7 @@ static TEE_Result mbedTLS_attest_key_cert(mbedtls_pk_context *issuer_key,
 	}
 
 	ret = mbedtls_x509write_crt_set_key_usage(&crt,
-					    MBEDTLS_X509_KU_DIGITAL_SIGNATURE |
-					    MBEDTLS_X509_KU_KEY_CERT_SIGN);
+					    key_usage);
```

Comment:
Key usage will never set MBEDTLS_X509_KU_KEY_CERT_SIGN here.
As result - this key can not be used for attestation (to sign certificates) due to [RFC 5280](https://tools.ietf.org/html/rfc5280#page-29)